playbooks/roles/cluster/tasks/workflow.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Configure Openshift authenticate job template
+  tower_job_template:
```

Comment:
This is creating a new job, rather than using the existing job created during the tower bootstrap.
tbh I'm in two minds.. Perhaps remove the one in the tower bootstrap section, and do it here instead?

Reply:
So this is because I specifically want to tower pod logged in as a limited user who can only access the specific tower project, not admin which the play will login as by default.

Reply:
Sounds good